src/ExpressionEvaluator/VisualBasic/Source/ExpressionCompiler/Binders/PlaceholderLocalBinder.vb

```diff
@@ -120,6 +128,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.ExpressionEvaluator
             End Select
         End Function
 
+        Friend Shared Function Canonicalize(name As String) As String
+            Return CaseInsensitiveComparison.ToLower(name)
+        End Function
```

Comment:
The dictionary is not case-insensitive because we want it to have the same behavior as the debugger-maintained dictionary, which is case-sensitive.